Gaurav.java

```diff
@@ -0,0 +1,28 @@
+import java.io.IOException;
+
+
+public class Gaurav {
+
+	 private static int count = 0;
+
+	
+	public static void main(String[] args) {
+		
+				System.out.println("This is Gaurav Garg testing using GIT");
+				
+				Process process =null;
+				
+				try {
+					process.getOutputStream().close();
+				} catch (IOException e) {
+					// TODO Auto-generated catch block
+					e.printStackTrace();
+				}
+	}
+	
+	public void doSomething() {
+	    //...
+	    count++;  // Noncompliant
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2696)